jax/_src/numpy/ufuncs.py

```diff
@@ -1585,8 +1585,49 @@ def _float_divmod(x1: ArrayLike, x2: ArrayLike) -> tuple[Array, Array]:
   return lax.round(div), mod
 
 
-@implements(np.power, module='numpy')
 def power(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  """Calculate element-wise base ``x1`` exponential of ``x2``.
+
+  JAX implementation of :obj:`numpy.power`.
+
+  Args:
+    x1: scalar or array. Specifies the bases.
+    x2: scalar or array. Specifies the exponent. ``x1`` and ``x2`` should either
+      have same shape or be broadcast compatible.
+
+  Returns:
+    An array containing the base ``x1`` exponentials of ``x2`` with same dtype
+    as input.
+
+  Note:
```

Comment:
We should add a note here about the lowering of this operation, because it's sometimes surprising. When `x2` is a concrete integer scalar, this lowers to `jax.lax.integer_pow`. When `x2` is a traced scalar or an array, this lowers to `jax.lax.pow`. It would be worth noting these two functions under `See also` as well.

Reply:
Updated the doc with suggested notes.